src/pytkdocs/loader.py

```diff
@@ -741,11 +741,19 @@ def get_django_field_documentation(node: ObjectNode) -> Attribute:
         if prop.blank:
             properties.append("blank")
 
+        # set correct docstring based on verbose_name and help_text
+        # both should be converted to str type in case lazy translation
+        # is being used, which is common scenario in django
+        if prop.help_text:
+            docstring = "`{}`: {}".format(prop.verbose_name, prop.help_text)
+        else:
+            docstring = "{}".format(prop.verbose_name)
```

Comment:
```suggestion
            docstring = prop.verbose_name
```

Reply:
If You won't cast it to string then You get serializer error with Proxy object that can't be serialized. Of course that only happens when `verbose_name` uses lazy translation (which is very common practice in django).

Reply:
Ah, my bad! Then a simple cast with str should suffice: `docstring = str(prop.verbose_name)`, no?

Reply:
I would go with format or f string which seems to be safer solution but str() should be ok in most cases.

Reply:
I don't think there's a difference, as f-strings and `.format` just call `__str__` on the object (happy to be proven wrong).

Reply:
You seems to be right. Somehow I've thought that format should do some magic with custom types but apparently it don't. So we can safely go with `docstring = str(prop.verbose_name)`.